src/coreclr/jit/gtlist.h

```diff
@@ -37,6 +37,7 @@ GTNODE(LABEL            , GenTree            ,0,0,GTK_LEAF)             // Jump-
 GTNODE(JMP              , GenTreeVal         ,0,0,GTK_LEAF|GTK_NOVALUE) // Jump to another function
 GTNODE(FTN_ADDR         , GenTreeFptrVal     ,0,0,GTK_LEAF)             // Address of a function
 GTNODE(RET_EXPR         , GenTreeRetExpr     ,0,0,GTK_LEAF|DBK_NOTLIR)  // Place holder for the return expression from an inline candidate
+GTNODE(GCPOLL           , GenTree            ,0,0,GTK_LEAF)
```

Comment:
```suggestion
GTNODE(GCPOLL           , GenTree            ,0,0,GTK_LEAF|DBK_NOTLIR)
```
Since this gets removed by rationalization